src/Components/CalendarComponent/CalendarComponent.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import CalendarComponent from './CalendarComponent';
+
+describe('CalendarComponent', () => {
+  it('should render a CalendarComponent with a title and ages', () => {
+    const { getByText } = render(<CalendarComponent />);
```

Comment:
We'll have to wrap this in the Apollo provider if we start making requests here

Reply:
Great catch, you're totally right!